.travis.yml

```diff
@@ -1,13 +1,26 @@
 language: java
-jdk:
-- oraclejdk8
 sudo: false
+# see https://sormuras.github.io/blog/2018-03-20-jdk-matrix.html
+matrix:
+  include:
+    # 8
+    - env: JDK='Oracle JDK 8'
+      jdk: oraclejdk8
+    - env: JDK='OpenJDK 8'
+      jdk: openjdk8
+    # 11
+    - env: JDK='Oracle JDK 11'
+      install: . ./install-jdk.sh -F 11 -L BCL
+    - env: JDK='OpenJDK 11'
+      install: . ./install-jdk.sh -F 11 -L GPL
 before_install:
+  - wget https://github.com/sormuras/bach/raw/master/install-jdk.sh
```

Comment:
What do you think about copying this script into our git repo, so it doesn't need to be pulled in every time?  Also prevents possible security issues if that script gets compromised for whatever reason (I know, I'm probably being overcautious :D).

Reply:
I actually tried both and went for the wget approach, thinking that it would be a bit nicer if we could avoid hosting that file on our repo. But your security concerns just convinced me of the opposite :)